File: app/src/main/java/com/shuneault/netrunnerdeckbuilder/game/CardCount.java
package com.shuneault.netrunnerdeckbuilder.game;

public class CardCount {
    private Card mCard;
    private int mCount;
    private boolean mDone;

    public CardCount(Card card, int count) {
        mCard = card;
        mCount = count;
        mDone = false;
    }

    public CardCount(Card card, int count, boolean done) {
        this(card, count);
        mDone = done;
    }

    public Card getCard() {
        return mCard;
    }

    public void setCard(Card mCard) {
        this.mCard = mCard;
    }

    public int getCount() {
        return mCount;
    }

    public void setCount(int mCount) {
        this.mCount = mCount;
    }

    public boolean isDone() {
        return mDone;
    }

    public void setDone(boolean done) {
        mDone = done;
    }
}

File: app/src/main/java/com/shuneault/netrunnerdeckbuilder/game/CardCount.kt
package com.shuneault.netrunnerdeckbuilder.game

class CardCount(var card: Card, var count: Int) {
    var isDone = false

    constructor(card: Card, count: Int, done: Boolean) : this(card, count) {
        isDone = done
    }

    fun add(quantity: Int) {
        count += quantity
    }
}